config/registry/google/modules/gcp-k8s-service.yaml

```diff
@@ -152,6 +154,11 @@ inputs:
       A list persistent storages to add to each instance of your service (need to give a `size` which is the size in GB
       for the storage volume to be, and `path` which is the path in the filesystem of each instance to place it under)
     default: []
+  - name: extra_project_iam_roles
```

Comment:
Can I create a role outside of opta and grant it specific permissions for a specific resource and then put it here? 
That way it should be possible to achieve granular permissions. No?

Reply:
How about we name this “additional_iam_roles”?

Reply:
You are confusing gcp roles with aws roles. gcp roles is more like aws iam policies